src/components/filters/FilterMenu.jsx

```diff
@@ -86,7 +86,13 @@ const FilterMenu = (props) => {
 
   // maps through queried data to get an array of start times to pass to selectedRange component
   const startDates =
-    data && data.events && data.events.map((event) => event.start)
+    data &&
+    data.events &&
+    data.events.map((event) => {
+      if (new Date(event.start) >= new Date()) {
```

Comment:
Good pass by value instead of reference here. I have been picking up this as a bug in a few places in our code and I have a feeling it still exists in other places